packages/cli/src/bin/pipcook.ts

```diff
@@ -45,6 +52,39 @@ export const cacheClean = async (opts: CacheCleanOptions): Promise<void> => {
   await Promise.all(futures);
 };
 
+export const createScriptRepo = async (scriptType: string, name: string): Promise<void> => {
```

Comment:
This function could be finished by `npm init`, see https://docs.npmjs.com/cli/v7/commands/npm-init/.

Reply:
Does that mean we need to maintain another project named `create-pipcook-script`?

Reply:
What we need is to rename our template to create-* projects.

Reply:
If that, the create-* projects must be node packages first? Does it support branch?